### PR TITLE
i18n(es): Add `core-concepts/astro-components.md`

### PR DESCRIPTION
**Pull request by agustinmulet**
- [x] Reviewed
- [x] Updated
- [x] Ready to go! 🧑‍🚀

### CODE REVIEW

File: src/pages/es/core-concepts/astro-components.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+---
+<style>
+  /* encapsulado en el componente, otros H1 en la página no estarán afectados */
+  h1 { color: red }
+</style>
+
+<h1>Hola mundo!</h1>
+```
+
+> ⚠️ Los estilos definidos aquí serán aplicados únicamente en la plantilla del cponente. **No** se verán afectados componentes hijos ni cualquier componente importado por defecto.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:342]:
```suggestion
> ⚠️ Los estilos definidos aquí serán aplicados únicamente en la plantilla del componente. **No** se verán afectados los componentes hijos ni cualquier otro componente importado por defecto.
```

**Review by kevinzunigacuellar — CHANGES_REQUESTED** [review]

File: src/pages/es/core-concepts/astro-components.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+layout: ~/layouts/MainLayout.astro
+title: Componentes
+description: Una introducción a la sintaxis de los componentes .astro.
+---
+
+**Los componentes Astro** son los bloques fundamentales de cualquier proyecto hecho con Astro. Son componentes de maquetado compuestos únicamente por HTML y sin ejecución del lado del cliente.
+
+La sintaxis de los componentes Astro es un superconjunto de HTML. La sintaxis fue [diseñada para que gente experimentada con HTML o JSX se sienta familiarizado](/es/comparing-astro-vs-other-tools/#astro-vs-jsx), y añade soporte para incluir componentes y expresiones de Javascript. Puedes identificar los componentes Astro por su extensión: `.astro`.
+
+Los componentes Astro son extremadamente flexibles. Muchas veces, un componente Astro va a contener **UI reutilizable en la página**, como puede ser un header o una profile card. Otras veces, un componente Astro puede contener un snippet pequeño de HTML, como puede ser una colección de etiquetas `<meta>` para facilitar nuestro trabajo con el SEO. Los componentes Astro también pueden contener el layout entero de una página.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:11]: Astro va a contener -> Astro puede contener

**Review by kevinzunigacuellar — APPROVED** [review]

**Comment by sarah11918** [comment]
Just updating the branch, because this is a few days old now, and checking in to see whether you all have come to an agreement on these suggestions!  :)

File: src/pages/es/core-concepts/astro-components.md (hunk 2 above)
Comment (readonlychild) [reply src/pages/es/core-concepts/astro-components.md:11]: de acuerdo con cambiar a "puede contener"

**Comment by kevinzunigacuellar** [comment]
LGTM 🚀

**Pull request merged**